lib/marc/controlfield.rb

```diff
@@ -26,7 +26,7 @@ def self.control_tag?(tag)
     # the value of the control field
     attr_accessor :value
 
-    # The constructor which must be passed a tag value and 
+    # The constructor which must be passed a tag value and
     # an optional value for the field.
 
     def initialize(tag,value='')
```

Comment:
Raise exception using input `tag` prior to any assignments for `@tag` and `@value`.  Perhaps this is a generic exception, i.e. http://ruby-doc.org/core-2.2.2/ArgumentError.html